<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderInterface</class>
 <widget class="QWidget" name="RenderInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1357</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
     <property name="spacing">
      <number>12</number>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>40</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="CardWidget" name="settingCard">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>400</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="settingVerticalLayout_1" stretch="0,0,0,1,0,1">
          <property name="spacing">
           <number>7</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="settingHorizontalLayout_1" stretch="0,0,0,0">
            <item>
             <widget class="IconWidget" name="widget" native="true">
              <property name="minimumSize">
               <size>
                <width>22</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>22</width>
                <height>22</height>
               </size>
              </property>
              <property name="icon" stdset="0">
               <iconset resource="../resource.qrc">
                <normaloff>:/images/render.png</normaloff>:/images/render.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="settingHorizontalSpacer_1">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>2</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="StrongBodyLabel" name="settingPeriodLabel">
              <property name="maximumSize">
               <size>
                <width>16151564</width>
                <height>200</height>
               </size>
              </property>
              <property name="text">
               <string>红外温度矩阵渲染</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="settingHorizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="settingVerticalLayout_2" stretch="0,0">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="StrongBodyLabel" name="settingModeLabel_1">
              <property name="text">
               <string>模式一: 单个矩阵渲染并展示</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="settingFileVerticalLayout"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="settingVerticalLayout_3">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="StrongBodyLabel" name="settingModeLabel_2">
              <property name="text">
               <string>模式二: 所有矩阵渲染并保存</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="settingDirVerticalLayout"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="CardWidget" name="irCard">
       <property name="minimumSize">
        <size>
         <width>380</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="irVerticalLayout_1" stretch="0,1">
          <item>
           <layout class="QHBoxLayout" name="irHorizontalLayout_1">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="IconWidget" name="irCardIcon_1" native="true">
              <property name="minimumSize">
               <size>
                <width>22</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>22</width>
                <height>22</height>
               </size>
              </property>
              <property name="icon" stdset="0">
               <iconset resource="../resource.qrc">
                <normaloff>:/images/camera.png</normaloff>:/images/camera.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="irHorizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>2</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="StrongBodyLabel" name="irPeriodLabel">
              <property name="text">
               <string>渲染结果</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="irHorizontalSpacer_1">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="irVerticalLayout_2">
            <item>
             <widget class="QLabel" name="irLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IconWidget</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StrongBodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
